torchaudio/models/wav2vec2/components.py

```diff
@@ -397,6 +401,28 @@ def forward(
 
         return x
 
+    def get_intermediate_outputs(
+            self,
+            x: Tensor,
+            indices: Optional[int] = None,
```

Comment:
this should be `Optional[List[int]]` instead as we expect a list of indices, not a single number. `len(indices)` would raise an error if `indices` is of type `int`

Reply:
The changes related to feature extraction has been moved to #1776 due to its BC-breaking change. I will address it there.